pages/get-started/introduction/why-magicblock.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Why MagicBlock?
+description: "The high-performance engine for real-time applications on Solana"
+---
+
+<Card>
+  MagicBlock is an extension of the Solana network designed for high-performance
+  decentralized applications. It enhances Solana’s capabilities while preserving
+  its composability and integrity.
+</Card>
+
+## Why MagicBlock?
+
+While blockchain technology is revolutionizing decentralized applications, it still faces fundamental challenges in:
+
+- **Latency** – Blockchain transaction speeds are too slow for real-time applications.
+- **Cost** – Even "low-fee" blockchains can become expensive at scale.
+- **Scalability** – Current architectures struggle to handle high-throughput applications.
+
+MagicBlock solves these issues with an **ephemeral rollup**, enabling developers to build dApps that require **10 ms state transitions**, **gasless transactions**, and **horizontal scaling**. Here's how:
+
+#### Built on Solana's Ecosystem and Performance
+
+MagicBlock operates as a specialized **Solana Virtual Machine (SVM) runtime**, seamlessly integrating with Solana’s base layer. Developers deploying programs on Solana are fully compatible with the ephemeral rollup down to the bytecode, benefiting from the SVM’s high performance and robust ecosystem.
+
+<img height="200" src="/images/magicblock-solana.png" />
+
+#### Overcoming Limitations in Scaling and Cost
+
+Traditional rollups and Solana’s current execution model have inherent constraints when handling resource-intensive applications. MagicBlock removes these limitations by enabling:
+
+- **Ultra-Low Latency (10 ms Block Time)**: Solana's default 400 ms block time is too slow for real-time applications. MagicBlock allows developers to customize execution environments while maintaining full compatibility.
```

Comment:
Since block time is arbitrarily configurable, I wonder if we should display a range instead, e.g. 1ms-10ms or 1ms-50ms

Reply:
Agreed, not sure what's the best way to communicate it. Maybe '< 10 ms' ? or ' < 50 ms' ?